.markdownlint-cli2.yaml

```diff
@@ -16,7 +16,7 @@ config:
 
   # We like to use really long lines
   line-length:
-    line_length: 400
+    line_length: 800
```

Comment:
to allow us to do a table with bigger content. See its comment ` # We like to use really long lines`. however, it was not long enough :-P

Reply:
Please do not change this value. 400 is already too long. I set it that way so that when I added the linter job I did not have to wrap every single line of every file in the repository, but we don't actually want it that long. Please wrap paragraphs to a reasonable width to make reviewing easier.

Reply:
@dhellmann, @jmrodri, 

A markdown table breaks the lines automatically and makes it easier for us to view the data.
However, the lint here is static and does not ignore tables. 
Also, I do not sure what is the motivation of this check. line_length: 400 requires to scroll such ass 500 such as 800. so, why it needs to be 400 and can't be 800?

Reply:
I chose 400 as the smallest value that allowed me to turn on the linter job as required without editing every file in the repository first. 80 or 100 would be my preference, but I haven't wanted to come back and edit all of those files to make that limit work. Raising the value to 800 makes it completely useless as a check and encourages authors to write documentation in a format that is hard to review, hard to track edits, etc.

It looks like the tables in this document could easily be turned into bullet lists and retain the semantic value of special formatting beyond regular paragraphs.

Reply:
linter rules that prevent people from using reasonable markdown structures seems like the tail wagging the dog.

Tables have headers and columns.  Bullet lists are not the same thing.

Reply:
Is there a way to flag some lines as "ignore"? Then we could flag the tables as ignore.